README.md

```diff
@@ -6,13 +6,38 @@ The SBOM tool is a highly scalable and enterprise ready tool to create SPDX 2.2
 
 ## Table of Contents
 
-* [Installation](docs/installation.md)
-* [Building and running Sbom tool](docs/build-and-run.md)
+* [Installation](#installation)
+* [Run the tool]()
+* [Building and running Sbom tool from source](docs/build-and-run.md)
```

Comment:
Isn't this linking back to the docs folder that contains the same instructions as "Run the tool" section

Reply:
No this is building from source, and I fixed the wording